python/requirements.txt

```diff
@@ -0,0 +1,14 @@
+click==8.1.7
+coloredlogs==15.0.1
+flatbuffers==23.5.26
+humanfriendly==10.0
+magika==0.5.0
```

Comment:
Why is magika here? This will override line 45 of the Dockerfile

Reply:
thanks you're right, i miss this using poetry
should be fixed